core/app/views/refinery/admin/_menu.html.erb

```diff
@@ -1,4 +1,25 @@
-<nav id="menu">
-  <%= render partial: "refinery/admin/menu_item",
-             collection: current_refinery_user.active_plugins.in_menu -%>
-</nav>
+<button type="button" class="btn btn-primary sidebar-left-opener fixed-t-l z-5 hidden-sm-up"><i class="icon icon-bars"></i></button>
```

Comment:
Button or link with icons only should have visuallyhidden text in it, like :

```ruby
<button type="button" class="btn btn-primary sidebar-left-opener fixed-t-l z-5 hidden-sm-up">
  <i class="icon icon-bars"></i>
  <span class="visuallyhidden">
    <%= t('.your_awesome_text') %>
  </span>
</button>
```

It's an WCAG best practice and easier to read the code.